modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -82,20 +82,24 @@ def status
         #
         # @return [JSON] Last POA change request through Claims API
         def active
-          power_of_attorney = ClaimsApi::PowerOfAttorney.find_using_identifier_and_source(header_md5: header_md5,
-                                                                                          source_name: source_name)
-          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Resource not found') unless power_of_attorney
-
-          if current_poa
-            lighthouse_poa = power_of_attorney.attributes
-            lighthouse_poa['current_poa'] = current_poa
-            lighthouse_poa['form_data'] = power_of_attorney.form_data
-            combined = ClaimsApi::PowerOfAttorney.new(lighthouse_poa)
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'POA not found') unless current_poa
 
-            render json: combined, serializer: ClaimsApi::PowerOfAttorneySerializer
-          else
-            render json: power_of_attorney, serializer: ClaimsApi::PowerOfAttorneySerializer
-          end
+          render json: {
+            data: {
+              id: nil,
+              type: 'claims_api_power_of_attorneys',
+              attributes: {
+                status: ClaimsApi::PowerOfAttorney::UPDATED,
+                date_request_accepted: 'N/A',
+                representative: {
+                  service_organization: {
+                    poa_code: current_poa
+                  }
+                },
+                previous_poa: 'N/A'
+              }
+            }
+          }
```

Comment:
I went this route, instead of a Serializer, so I could be extremely explicit with the structure.
I can revise if this is unacceptable.